src/audiomixerboard.h

```diff
@@ -182,65 +196,68 @@ class CAudioMixerBoardSlots : public CAudioMixerBoardSlots<slotId - 1>
 };
 
 template<>
-class CAudioMixerBoardSlots<0> {};
+class CAudioMixerBoardSlots<0>
```

Comment:
Single line preferred in this situation, if that's possible.  (i.e. empty body, not too wide)

Reply:
fixed that by adding
```
AllowShortBlocksOnASingleLine: Empty
```

Reply:
Seems to have reverted?

Reply:
Hm - is this another v12 feature?